app/models/ability.rb

```diff
@@ -128,7 +128,10 @@ def owner_perms
         teams << v_obj.team_id if teams.blank? and v_obj.respond_to?(:team)
         teams << v_obj.project.team_id if teams.blank? and v_obj.respond_to?(:project)
       end
-      teams << v_obj_parent.project.team_id if v_obj_parent and v_obj_parent.respond_to?(:project)
+      if v_obj_parent
+        teams << v_obj_parent.team_id if v_obj_parent.respond_to?(:team)
+        teams << v_obj_parent.project.team_id if v_obj_parent.respond_to?(:project)
```

Comment:
I would prefer bypassing project completely to find the team

Reply:
Done